fix(search): Add description for "level"

JoshFerge opened the pull request:
was missing, so added one!

doralchan commented:
The format we have is:

Description (i.e., example1, example2)

so perhaps:

Severity of the event (i.e., Fatal, Critical, Warning)

![Screenshot 2023-01-11 at 11 48 16 AM](https://user-images.githubusercontent.com/4830259/211903506-8b068ec3-c770-41fe-9e87-b45df47cd98d.png)

JoshFerge commented:
> The format we have is:
> 
> Description (i.e., example1, example2)
> 
> so perhaps:
> 
> Severity of the event (i.e., Fatal, Critical, Warning)
> 
> ![Screenshot 2023-01-11 at 11 48 16 AM](https://user-images.githubusercontent.com/4830259/211903506-8b068ec3-c770-41fe-9e87-b45df47cd98d.png)

updated. also i see there's a keywords field, which i've added. should that be kept?

doralchan reviewed (approved):
Looks good to me!
